src/util/requestDispatcher.js

```diff
@@ -115,4 +129,33 @@ export default class RequestDispatcher {
     }
     return url
   }
+
+  getDelayFrequency () {
+    this.logger.debug('Network retry #' + this.networkRetryCount)
+
+    // Retry with delay as 1s for first 10 retries
+    if (this.networkRetryCount < 10) {
+      this.logger.debug(this.account.id, 'Failure count is ' + this.networkRetryCount + '. Setting delay frequency to 1s')
+      this.minDelayFrequency = PUSH_DELAY_MS // Reset minimum delay to 1s
+      return this.minDelayFrequency
+    }
+
+    if (this.account.region == null) {
```

Comment:
===

Reply:
== is correct, as it will check for both null and undefined